14_Day_Higher_order_functions/14_higher_order_functions.md

```diff
@@ -180,11 +180,12 @@ def split_string_decorator(function):
 
     return wrapper
 
+#Decorators will be executed from bottom to top
 @split_string_decorator
-@uppercase_decorator     # order with decorators is important in this case - .upper() function does not work with lists
```

Comment:
[doubt]

why did you remove this comment ?

i believe that "#.upper() function does not work with lists" is interesting, what you think?